<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-05-15T12:36:47 IST" name="Suite" finished-at="2023-05-15T12:37:19 IST" duration-ms="31547">
    <groups>
    </groups>
    <test started-at="2023-05-15T12:36:47 IST" name="Test" finished-at="2023-05-15T12:37:19 IST" duration-ms="31547">
      <class name="TestCase.VerifyLoginTestCase">
        <test-method is-config="true" signature="launchbrowser()[pri:0, instance:TestCase.VerifyLoginTestCase@61862a7f]" started-at="2023-05-15T12:36:47 IST" name="launchbrowser" finished-at="2023-05-15T12:36:53 IST" duration-ms="5963" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchbrowser -->
        <test-method signature="login()[pri:0, instance:TestCase.VerifyLoginTestCase@61862a7f]" started-at="2023-05-15T12:36:53 IST" name="login" finished-at="2023-05-15T12:36:55 IST" duration-ms="1673" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="quitebrowser()[pri:0, instance:TestCase.VerifyLoginTestCase@61862a7f]" started-at="2023-05-15T12:36:55 IST" name="quitebrowser" finished-at="2023-05-15T12:36:55 IST" duration-ms="327" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitebrowser -->
      </class> <!-- TestCase.VerifyLoginTestCase -->
      <class name="TestCase.VerifySignUpTestCase">
        <test-method is-config="true" signature="launchbrowser()[pri:0, instance:TestCase.VerifySignUpTestCase@dd8ba08]" started-at="2023-05-15T12:36:55 IST" name="launchbrowser" finished-at="2023-05-15T12:37:00 IST" duration-ms="4039" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchbrowser -->
        <test-method signature="SignUp()[pri:0, instance:TestCase.VerifySignUpTestCase@dd8ba08]" started-at="2023-05-15T12:37:00 IST" name="SignUp" finished-at="2023-05-15T12:37:19 IST" duration-ms="19018" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [ CRM Trial: Salesforce 30-Day Trial - Salesforce IN] but found [Free CRM Trial: Salesforce 30-Day Trial - Salesforce IN]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [ CRM Trial: Salesforce 30-Day Trial - Salesforce IN] but found [Free CRM Trial: Salesforce 30-Day Trial - Salesforce IN]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Resources.CommonUtilities.assertionhandle(CommonUtilities.java:22)
at TestCase.VerifySignUpTestCase.SignUp(VerifySignUpTestCase.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SignUp -->
        <test-method is-config="true" signature="quitebrowser()[pri:0, instance:TestCase.VerifySignUpTestCase@dd8ba08]" started-at="2023-05-15T12:37:19 IST" name="quitebrowser" finished-at="2023-05-15T12:37:19 IST" duration-ms="445" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitebrowser -->
      </class> <!-- TestCase.VerifySignUpTestCase -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
